app/models/user.rb

```diff
@@ -203,9 +203,11 @@ def unread_inbox_comments_count
                                    max_pwd: ArchiveConfig.PASSWORD_LENGTH_MAX)
 
   validates_format_of :login,
-                      message: ts("must be 3 to 40 characters (A-Z, a-z, _, 0-9 only), no spaces, cannot begin or end with underscore (_)."),
+                      message: ts("^User name must be %{min_pwd} to %{max_pwd} characters (A-Z, a-z, _, 0-9 only), no spaces, cannot begin or end with underscore (_).",
+                                  min_pwd: ArchiveConfig.LOGIN_LENGTH_MIN,
```

Comment:
Can you change `pwd` to `login` here, since we're describing the login length, not password length?

Reply:
Got it — thanks for catching that 😅